examples/gnrc_border_router/README.md

```diff
@@ -1,4 +1,159 @@
-# gnrc_networking_border_router example
+# gnrc_border_router using automatic configuration
+This setup uses a single serial interface, ethos (Ethernet Over Serial) 
+and UHCP (micro Host Configuration Protocol). 
+Ethos multiplexes serial data to separate ethernet packets from shell commands.
+UHCP is in charge of configuring the wireless interface prefix 
+and routes on the BR.
+
+The script `start_network.sh` enables a *ready-to-use* BR in only one command.
+
+## Requirements
+This functionality works only on Linux machines.
+Mac OSX support will be added in the future (lack of native `tap` interface).
+
+## Setup
+First, you need to compile `ethos`.
+Go to `/dist/tools/ethos` and type:
+
+```bash
+make clean all
+```
+
+Then, you need to compile UHCP.
+This tool is found in `/dist/tools/uhcp`. So, as for `ethos`:
+
+```bash
+make clean all
+```
+
+Afterwards, proceed to compile and flash `gnrc_border_router` to your board:
+
+```bash
+make clean all flash
+```
+
+## Usage
+Start the `start_network.sh` script by doing on `dist/tools/ethos`:
+
+```bash
+sudo sh start_network.sh /dev/ttyACMx tap0 2001:db8::/64
+```
+
+This will execute the needed commands to setup a `tap` interface 
+and configure the BR.
+Notice that this will also configure `2001:db8::/64` as a prefix.
+This prefix should be announced to other motes through the wireless interface.
+
+As said previously, `ethos` allows to send IP packets and shell commands.
+This is done through the same serial interface.
+By typing `help` you will get the list of available shell commands.
+
+At this point you should be able to ping motes using the global address.
+For instance, if you use the `gnrc_networking` example on the mote, you can 
+ping it from your machine with:
+
+```
+> ping6 2001:db8:0:1234:0:567:8:1
+```
+
+Just replace this address by your mote's address.
+Using `ifconfig` on the shell of your mote shows you the addresses of your 
+mote.
+
+The script also sets up a ULA (Unique Local Address) address on your Linux 
+network interface.
+You check your ULA on your PC with `ifconfig` Linux command.
+It can be pinged from 6lo motes:
+
+```
+> ping6 fd00:dead:beef::1
+```
+
+Thus far, IPv6 communication with between your PC and your motes is enabled.
+
+## Issues
+A known issue [#5122](https://github.com/RIOT-OS/RIOT/issues/5122) 
+is that after ~15min global addresses on the BR Neighbour Cache dissappear.
+We are investigating why it happens and we'll propose a solution soon.
+If you experience it, just reboot your nodes and they will come again on-line.
+
+# gnrc_border_router with manual config
```

Comment:
I guess it's fine. However: let's merge that asap, and also put it in the release candidate branch so that people have the required documentation to test...